editor/src/clj/editor/disk.clj

```diff
@@ -174,11 +174,10 @@
         true)
 
     exception
-    (if (engine-build-errors/handle-build-error! render-error! project evaluation-context exception)
-      true
-      (throw exception))))
+    (do (engine-build-errors/handle-build-error! render-error! project evaluation-context exception)
+        true)))
```

Comment:
I think the original intent here was that we'd get an exception if we got some kind of error we weren't expecting so that we could add proper support for it in `handle-build-error!`. But maybe somehow we're not showing the error dialog?

I guess it's fine to simply just post the error in the build errors view anyway. We'll hopefully still get a feature request from our users if the error lacks information.